hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/ClientPartitionServiceImpl.java

```diff
@@ -231,10 +231,13 @@ public int getPartitionId() {
         @Override
         public Member getOwner() {
             Address owner = getPartitionOwner(partitionId);
-            if (owner != null) {
-                return client.getClientClusterService().getMember(owner);
+            if (owner == null) {
+                ClientMessage message = ClientTriggerPartitionAssignmentCodec.encodeRequest();
+                ClientInvocation invocation = new ClientInvocation(client, message, null);
+                invocation.invoke();
```

Comment:
Do we need an urgent invoke here?

Reply:
I don't think we need urgent invoke. This is something that will work eventually anyway.